imagely.bin.js

```diff
@@ -17,6 +22,41 @@ options.bg = args.bg || args.b;
 options.json = args.json || args.d;
 options.log = args.log || args.l;
 options.batch = args.batch;
+options.logFilepath = args.logFilepath;
+
+function addToBatchLogs(json) {
+	var log;
+	var dimensions;
+	var imageKey = json[0].key;
+	var imageKeyArray = imageKey.split('_');
+
+	try {
+		dimensions = imageSize(destination);
+	}
+	catch (exception) {
+		dimensions = { width: null, height: null };
+	}
+
+	log = dimensions;
+	log['uuid'] = imageKeyArray[1];
+	log['endpoint'] = imageKeyArray[0];
```

Comment:
I think this whole commit is very site specific (hence commit message), which sort of brings up a broader point of maybe having this be in a separate branch, or passing in the callback altogether from the code base. I feel like this file should literally just be setting options and instantiating Imagely with options and a user set callback.

Not sure the cleanest way to do this.